templates/CommentsInterface.ss

```diff
@@ -1,29 +1,18 @@
 <% if CommentsEnabled %>
+	<%	require javascript('comments/javascript/commenting.js') %>
+	<%  require css('comments/css/comments.css') %>
```

Comment:
Also <% require %> in a condition doesn't actually work. If you need to conditionally require it it has to go into php.